crates/nu-protocol/src/parse_warning.rs

```diff
@@ -0,0 +1,23 @@
+use crate::Span;
+use miette::Diagnostic;
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
+#[derive(Clone, Debug, Error, Diagnostic, Serialize, Deserialize)]
+pub enum ParseWarning {
+    #[error("Deprecated: {0}")]
+    DeprecatedWarning(
+        String,
+        String,
+        #[label = "`{0}` is deprecated. Please use `{1}` instead, more info: https://www.nushell.sh/book/custom_commands.html"]
```

Comment:
i would add the release from which the feature should be removed here, maybe 0.90?

Reply:
Yeah I think so.  I can remove it on 0.90

Reply:
nice :ok_hand: 

actually, what i meant was the following change, making it explicit that the annotation support will be removed in a given version :wink: 
```suggestion
        #[label = "`{0}` is deprecated and will be removed in 0.90. Please use `{1}` instead, more info: https://www.nushell.sh/book/custom_commands.html"]
```